### PR TITLE
Sync Patch 44356 & translate to chinese

### PR DESCRIPTION
**Pull request by mahmut-Abi**
sync patch #44356 , and transplate to chinese

### CODE REVIEW

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md [hunk 1]
```diff
@@ -399,17 +398,15 @@ The container runtime to use. Possible values: <code>docker</code>, <code>remote
...
+The endpoint of remote runtime service. UNIX domain sockets are supported on Linux, while 'npipe' and 'tcp' endpoints are supported on windows. Examples: <code>'unix:///path/to/runtime.sock'</code>, <code>'npipe:////./pipe/runtime'</code>. (DEPRECATED: This parameter should be set via the config file specified by the kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)
 -->
-远程运行时服务的端点。目前支持 Linux 系统上的 UNIX 套接字和
-Windows 系统上的 npipe 和 TCP 端点。例如：
-<code>unix:///path/to/runtime.sock</code>、
-<code>npipe:////./pipe/runtime</code>。
+
+远程运行时服务的终端。在 Linux 上支持 UNIX 域套接字，而在 Windows 上支持 'npipe' 和 'tcp' 终端。示例：<code>'unix:///path/to/runtime.sock'</code>，<code>'npipe:////./pipe/runtime'</code>。（已弃用：应该通过 kubelet 的 <code>--config</code> 标志指定的配置文件进行设置。请参阅 <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> 了解更多信息。）
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:409]:
```suggestion
远程运行时服务的终端。在 Linux 上支持 UNIX 域套接字，而在 Windows 上支持 'npipe' 和 'tcp' 终端。
示例：<code>'unix:///path/to/runtime.sock'</code>，<code>'npipe:////./pipe/runtime'</code>。
（已弃用：应该通过 kubelet 的 <code>--config</code> 标志指定的配置文件进行设置。请参阅
<a href="https://kubernetes.io/zh-cn/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> 了解更多信息。）
```

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md [hunk 2]
```diff
@@ -284,11 +285,9 @@ The path to the cloud provider configuration file. Empty string for no configura
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-The provider for cloud services. Set to empty string for running with no cloud provider. If set, the cloud provider determines the name of the node (consult cloud provider documentation to determine if and how the hostname is used). (DEPRECATED: will be removed in 1.24 or later, in favor of removing cloud provider code from kubelet.)
+The provider for cloud services. Set to empty string for running with no cloud provider. Set to 'external' for running with an external cloud provider. If set, the cloud provider determines the name of the node (consult cloud provider documentation to determine if and how the hostname is used).
 -->
-云服务的提供者。设置为空字符串表示在没有云驱动的情况下运行。
-如果设置了此标志，则云驱动负责确定节点的名称（参考云提供商文档以确定是否以及如何使用主机名）。
-已弃用：将在 1.24 或更高版本中移除，以便于从 kubelet 中去除云驱动代码。
+云服务的提供者。设置为空字符串表示在没有云服务提供商的情况下运行。设置为 'external' 表示在使用外部云服务提供商的情况下运行。如果设置，云服务提供商将确定节点的名称（请参考云服务提供商文档，以确定主机名是否被使用以及如何使用）。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:290]:
```suggestion
云服务的提供者。设置为空字符串表示在没有云驱动的情况下运行。
设置为 'external' 表示在使用外部云驱动的情况下运行。如果设置了此标志，
则云驱动负责确定节点的名称（参考云提供商文档以确定是否以及如何使用主机名）。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md [hunk 3]
```diff
@@ -1138,7 +1143,7 @@ The percent of disk usage before which image garbage collection is never run. Lo
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-The endpoint of remote image service. If not specified, it will be the same with <code>--container-runtime-endpoint</code> by default. UNIX Domain Socket are supported on Linux, while npipe and TCP endpoints are supported on Windows.  Examples: <code>unix:///var/run/dockershim.sock</code>, <code>npipe:////./pipe/dockershim</code>
+The endpoint of remote image service. If not specified, it will be the same with <code>--container-runtime-endpoint</code> by default. UNIX domain socket are supported on Linux, while `npipe` and `tcp` endpoints are supported on Windows. Examples: <code>unix:///path/to/runtime.sock</code>, <code>npipe:////./pipe/runtime</code>. (DEPRECATED: This parameter should be set via the config file specified by the kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:1146]: Update the zh text accordingly please

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md [hunk 4]
```diff
@@ -1195,7 +1200,7 @@ If enabled, the kubelet will integrate with the kernel memcg notification to det
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-Burst to use while talking with kubernetes API server. The number must be >= 0. If 0 will use default burst (10). (DEPRECATED: This parameter should be set via the config file specified by the kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)
+Burst to use while talking with the Kubernetes API server. The number must be >= 0. If 0, will use default burst (100). Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags. (DEPRECATED: This parameter should be set via the config file specified by the kubelet's <code>--config</code> flag. See <a href="/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:1203]: Update the zh text accordingly please

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md [hunk 5]
```diff
@@ -1235,7 +1240,7 @@ QPS to use while talking with kubernetes API server. The number must be &gt;= 0.
 </tr>
 
 <tr>
-<td colspan="2">--kube-reserved string&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<!--Default: &lt;None&gt;-->默认值：&lt;None&gt;</td>
+<td colspan="2">--kube-reserved strings&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: &lt;None&gt;</td>
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:1243]: `default:` can be translated too

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md [hunk 6]
```diff
@@ -1728,7 +1733,7 @@ kubelet 默认值不同时，kubelet 都会出错。
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-Unique identifier for identifying the node in a machine database, i.e cloud provider. (DEPRECATED: This parameter should be set via the config file specified by the kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)
+Unique identifier for identifying the node in a machine database, i.e cloud provider.
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:1736]: Update the zh text accordingly please

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md [hunk 7]
```diff
@@ -2250,12 +2255,9 @@ The full path of the directory in which to search for additional third party vol
...
+Specifies interval for kubelet to calculate and cache the volume disk usage for all pods and volumes. To disable volume calculations, set to a negative number. (DEPRECATED: This parameter should be set via the config file specified by the kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)
 -->
-指定 kubelet 计算和缓存所有 Pod 和卷的磁盘用量总值的时间间隔。要禁用磁盘用量计算，
-可设置为 0。
-（已弃用：应在 <code>--config</code> 所给的配置文件中进行设置。
-请参阅 <a href="https://kubernetes.io/zh-cn/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> 了解更多信息。）
+指定 kubelet 计算和缓存所有 pod 和卷的卷磁盘使用情况的间隔。要禁用卷计算，请将其设置为负数。（已弃用：应该通过 kubelet 的 <code>--config</code> 标志指定的配置文件进行设置。请参阅 <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> 了解更多信息。）
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:2260]: Follow the original zh text please if you don't understand the text really.

**Review by windsonsea — COMMENTED** [review]
It's recommended to localize the language you are familiar with.

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md (hunk 3 above)
Comment (mahmut-Abi) [reply content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:1148]: has been translated

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md [hunk 8]
```diff
@@ -271,10 +271,11 @@ If set, any request presenting a client certificate signed by one of the authori
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-The path to the cloud provider configuration file. Empty string for no configuration file. (DEPRECATED: will be removed in 1.24 or later, in favor of removing cloud providers code from kubelet.)
+The path to the cloud provider configuration file. Empty string for no configuration file. (DEPRECATED: will be removed in 1.25 or later, in favor of removing cloud providers code from kubelet.)
 -->
-云驱动配置文件的路径。空字符串表示没有配置文件。
-已弃用：将在 1.24 或更高版本中移除，以便于从 kubelet 中去除云驱动代码。
+
+云服务提供商配置文件的路径。空字符串表示没有配置文件。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:277]:
```suggestion
云驱动配置文件的路径。空字符串表示没有配置文件。
```
Follow the origin zh text. Original is better. Please check through.

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md [hunk 9]
```diff
@@ -271,10 +271,11 @@ If set, any request presenting a client certificate signed by one of the authori
...
+（已弃用：将在 1.25 版本或之后移除，以便删除 kubelet 中的云服务提供商代码。）
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:278]:
```suggestion
（已弃用：将在 1.25 或更高版本中移除，以便于从 kubelet 中去除云驱动代码。）
```

**Review by windsonsea — COMMENTED** [review]
尽量留用原来的中文，除非它是错误的。

btw: squash your commits into a single PR please.

**Comment by mahmut-Abi** [comment]
I have merged those commit.

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md (hunk 1 above)
Comment (windsonsea) [reply content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:411]: You can copy and paste the suggestions.

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md [hunk 10]
```diff
@@ -696,7 +695,7 @@ Whether kubelet should exit upon lock-file contention.
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-When set to <code>true</code>, hard eviction thresholds will be ignored while calculating node allocatable. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/">here</a> for more details. (DEPRECATED: will be removed in 1.24 or later)
+When set to <code>true</code>, hard eviction thresholds will be ignored while calculating node allocatable. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/reserve-compute-resources/">here</a> for more details. (DEPRECATED: will be removed in 1.25 or later)
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:698]: update the zh text too

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md [hunk 11]
```diff
@@ -1138,11 +1145,9 @@ The percent of disk usage before which image garbage collection is never run. Lo
...
+The endpoint of remote image service. If not specified, it will be the same with <code>--container-runtime-endpoint</code> by default. UNIX domain socket are supported on Linux, while `npipe` and `tcp` endpoints are supported on Windows. Examples: <code>unix:///path/to/runtime.sock</code>, <code>npipe:////./pipe/runtime</code>. (DEPRECATED: This parameter should be set via the config file specified by the kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)
 -->
-远程镜像服务的端点。若未设定则默认情况下使用 <code>--container-runtime-endpoint</code>
-的值。目前支持的类型包括在 Linux 系统上的 UNIX 套接字端点和 Windows 系统上的 npipe 和 TCP 端点。
-例如：<code>unix:///var/run/dockershim.sock</code>、<code>npipe:////./pipe/dockershim</code>。
+远程镜像服务的终端。如果未指定，将默认与 <code>--container-runtime-endpoint</code> 相同。在 Linux 上支持 UNIX 域套接字，而在 Windows 上支持 `npipe` 和 `tcp` 终端。示例：<code>unix:///path/to/runtime.sock</code>，<code>npipe:////./pipe/runtime</code>。（已弃用：应该通过 kubelet 的 <code>--config</code> 标志指定的配置文件进行设置。请参阅 <a href="https://kubernetes.io/zh-cn/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> 了解更多信息。）
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:1150]: The original text is better. You only need to translate those changed text.

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md [hunk 12]
```diff
@@ -1195,12 +1200,9 @@ If enabled, the kubelet will integrate with the kernel memcg notification to det
...
+Burst to use while talking with the Kubernetes API server. The number must be >= 0. If 0, will use default burst (100). Doesn't cover events and node heartbeat apis which rate limiting is controlled by a different set of flags. (DEPRECATED: This parameter should be set via the config file specified by the kubelet's <code>--config</code> flag. See <a href="/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)
 -->
-每秒发送到 API 服务器 的突发请求数量上限。
-该数字必须大于或等于 0。如果为 0，则使用默认的突发值（100）。
-（已弃用：应在 <code>--config</code> 所给的配置文件中进行设置。
-请参阅 <a href="https://kubernetes.io/zh-cn/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> 了解更多信息。）
+与 Kubernetes API 服务器通信时使用的突发速率。该数字必须大于或等于 0。如果为 0，则使用默认的突发速率（100）。不包括事件和节点心跳 API，其速率限制由一组不同的标志控制。（已弃用：应该通过 kubelet 的 <code>--config</code> 标志指定的配置文件进行设置。请参阅 <a href="/zh-cn/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> 了解更多信息。）
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:1205]: Follow the original zh text style please.

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md [hunk 13]
```diff
@@ -1728,11 +1730,9 @@ kubelet 默认值不同时，kubelet 都会出错。
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-Unique identifier for identifying the node in a machine database, i.e cloud provider. (DEPRECATED: This parameter should be set via the config file specified by the kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)
+Unique identifier for identifying the node in a machine database, i.e cloud provider.
 -->
-设置主机数据库（即，云驱动）中用来标识节点的唯一标识。
-（已弃用：应在 <code>--config</code> 所给的配置文件中进行设置。
-请参阅 <a href="https://kubernetes.io/zh-cn/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> 了解更多信息。）
+在机器数据库中标识节点的唯一标识符，即云服务提供商。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:1735]:
```suggestion
在主机数据库（即云驱动）中标识节点的唯一标识符。
```
Do you really understand the original zh text?

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md [hunk 14]
```diff
@@ -2212,9 +2212,9 @@ number for the log level verbosity
 <tr>
 <td></td><td style="line-height: 130%; word-wrap: break-word;">
 <!--
-Print version information and quit; --version=vX.Y.Z... sets the reported version.
+Print version information and quit; <code>--version=vX.Y.Z...</code> sets the reported version.
 -->
-打印 kubelet 版本信息并退出；--version=vX.Y.Z... 设置报告的版本。
+打印版本信息并退出；<code>--version=vX.Y.Z...</code> 设置报告的版本。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:2217]:
```suggestion
打印 kubelet 版本信息并退出；<code>--version=vX.Y.Z...</code> 设置报告的版本。
```

File: content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md [hunk 15]
```diff
@@ -2250,12 +2250,9 @@ The full path of the directory in which to search for additional third party vol
...
+Specifies interval for kubelet to calculate and cache the volume disk usage for all pods and volumes. To disable volume calculations, set to a negative number. (DEPRECATED: This parameter should be set via the config file specified by the kubelet's <code>--config</code> flag. See <a href="https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> for more information.)
 -->
-指定 kubelet 计算和缓存所有 Pod 和卷的磁盘用量总值的时间间隔。要禁用磁盘用量计算，
-可设置为 0。
-（已弃用：应在 <code>--config</code> 所给的配置文件中进行设置。
-请参阅 <a href="https://kubernetes.io/zh-cn/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> 了解更多信息。）
+指定 kubelet 计算和缓存所有 Pod 和卷的磁盘用量总值的时间间隔。要禁用磁盘用量计算， 可将其设置为负数。（已弃用：应该通过 kubelet 的 <code>--config</code> 标志指定的配置文件进行设置。请参阅 <a href="https://kubernetes.io/zh-cn/docs/tasks/administer-cluster/kubelet-config-file/">kubelet-config-file</a> 了解更多信息。）
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/kubelet.md:2255]: follow the original zh text style please.

**Review by windsonsea — COMMENTED** [review]
I don't think you really understand the zh text.

Please read through [中文本地化样式指南](https://kubernetes.io/zh-cn/docs/contribute/localization_zh/).

/hold

**Comment by windsonsea** [comment]
Thanks for your contribution though this will be closed due to much issues included.
/close

**Pull request closed without merging**